code/weapon/weapons.cpp

```diff
@@ -1503,9 +1506,15 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 	}
 
 	if(optional_string("$Free Flight Speed:")) {
-		float temp;
-		stuff_float(&temp);
-		nprintf(("Warning", "Ignoring free flight speed for weapon '%s'\n", wip->name));
+		stuff_float(&(wip->free_flight_speed));
+		if (wip->free_flight_speed < 0.01f) {
+			nprintf(("Warning", "Free Flight Speed value is too low for weapon '%s'. Resetting to default (0.25 times maximum)\n", wip->name));
+			wip->free_flight_speed = 0.25f;
```

Comment:
Is this the same value as `HOMING_DEFAULT_FREE_FLIGHT_FACTOR`? If so, maybe that could be used here to reduce the usage of magic numbers.